html2docx/html2docx.py

```diff
@@ -106,7 +121,7 @@ def finish_run(self) -> None:
 
     def add_text(self, data: str) -> None:
         if self.p is None:
-            style = self.list_style[-1] if self.list_style else None
+            style = self.list_style[-1] if self.list_style else self.style
```

Comment:
Does this mean lists and quotes are mutually exclusive? How is the following HTML handled? 

```
<ul><li><blockquote>lorem ipsum</blockquote></li></ul>
```

The above was generated using TinyMCE.

We'll want to support nested styles if possible. If this is not possible, it might be worth adding a "Limitations" section to the README. Could you please include it as a test as well?

Reply:
The docx package offers a choice of any standard style registered for the document. "Quote" and "List Bullet" are two such styles. To tweak a paragraph style beyond what a registered style offers means an addition of some attributes that are not part of a style.
This is something that docx package does not offer out of the box.